src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -84,6 +84,12 @@ public void updatePerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
     }
 
     @Override
+    public void sortPersons() {
```

Comment:
This requires a Javadoc comment. 
(If i accidentally deleted it during the merge, sorry! ><)

Reply:
ok ill get to it

Reply:
There wasn't any javdoc comments on the surrounding @Override methods as well. It also passes current gradlew checkStyleTest and checkStyleMain.

Reply:
I think it could be due to he removeTag method not this override method lemme get it fix